patches/chrome-common-importer-profile_import.typemap.patch

```diff
@@ -1,26 +1,28 @@
 diff --git a/chrome/common/importer/profile_import.typemap b/chrome/common/importer/profile_import.typemap
-index 6283f2bf6871a10f710694772b5da0bc9b70c2ad..1c0b74b2fc728e07fb59adc5c0f5ed257900357c 100644
+index 6283f2bf6871a10f710694772b5da0bc9b70c2ad..abd035b2d2a9db4dfb6834572404eb0a45d4b398 100644
 --- a/chrome/common/importer/profile_import.typemap
 +++ b/chrome/common/importer/profile_import.typemap
-@@ -4,6 +4,7 @@
+@@ -4,6 +4,8 @@
  
  mojom = "//chrome/common/importer/profile_import.mojom"
  public_headers = [
++  "//brave/common/importer/brave_ledger.h",
```

Comment:
and again here, even if we couldn't combine the data, we should certainly combine the header files so we only have to patch once. We should always be looking for places where we can create extension points that allow us to make additional changes without additional patches